### ISSUE TITLE
Consistent model naming / referring

### ISSUE THREAD

**Issue by sprypradeep** [jonathan-casarrubias/loopback-sdk-builder#50]
Although the models and APIs are referred to in Title case, the interface and models are being exported in provided case (in my case lower case). This results in errors in case you have any model name in lower case.

Following two changes correct the Title casing for properties and models: (utils.js)
```

    var name = c.name;
    var name = c.name[0].toUpperCase() + c.name.slice(1);
```

```
 targetClass = targetClass[0].toUpperCase() + targetClass.slice(1);
  modelClass.sharedClass.ctor.relations[scopeName].targetClass = targetClass;

```

**Comment by sprypradeep** [comment]
The issue still appears since index.ejs refer to models/ services in Title case.

**Comment by jonathan-casarrubias** [comment]
This issue is already fixed, will be published in pre-release 2.0.0-rc.5

**Comment by sprypradeep** [comment]
neat
